### ISSUE TITLE
signatureLocal: Cannot resolve signature;3 ff 72.0.1

### ISSUE THREAD

**Issue by MM-John** [feller-prj/extractor-project#19]
First thanks for the best video downloader on Firefox.

Now, I got this error on Firefox 72.0.1.

> signatureLocal: Cannot resolve signature;3

It was working before but I don't know what version.

This is on youtube. Screenshot attached
![failure_capture](https://user-images.githubusercontent.com/13987991/73128487-1ea90f00-3f9e-11ea-9ef7-099fc03cfca0.PNG)

This is mentioned in the reviews: https://addons.mozilla.org/en-US/firefox/addon/youtube_downloader_webx/reviews/
starting 01/24/2020 12:04pm EST

Here's a link to the first report chronologically in reviews: https://addons.mozilla.org/en-US/firefox/addon/youtube_downloader_webx/reviews/1480613/

Today, it was reported to have started working, his timestamp of his update: 1-25-2020 11:25AM Eastern. But others after still have the problem. Review who reported update is working: https://addons.mozilla.org/en-US/firefox/addon/youtube_downloader_webx/reviews/1480880/

I did just update firefox to 72.0.2 and had the same issue.

There's a similar error reported in this issue: https://github.com/feller-prj/extractor-project/issues/5
There it was about addon-signing and certificates.

uBlock origin (for an example of another addon) seems to be working fine.

**Comment by Richard996** [comment]
Yup!  Youtube has really cracked down, a day or two ago yD was doing it's usual excellent job and the next day the signature problem is stopping a lot of downloads.  Like the QuiteFrankly channel which is sort of consertaive political commentary with some usually short clips of music, and the YT signature issue has stopped me from downloading that site.  There was another way I used to get videos that were hard to download "www,clipconverter.cc" was good & I tried to dl a qf video and it wouldn't do it because it sited a music copyright issue, this site suggested I install the "Airy" video downloader (I might be off on the spelling) but I'm not keen on installing anything because let the downloader beware..
     So my guess this is YouTube smacking down copyright content, and I guess they have to.  Wouldn't they be legally liable if they let people download copyrighted material.  It was very convenient to be able to download videos and view them later, but YT seems to be making it difficult or impossible.  It's a pain for content that just has very short clips of music and they get shut down too.  That's my guess.  Maybe yD can overcome this signature issue?
